release-doc-in-process/all_vf.TTL

```diff
@@ -6,255 +6,854 @@
+        rdfs:comment        "Reference an agreement between agents which specifies the rules or policies which govern this commitment." .
+
+vf:inExchangeFor  a         owl:ObjectProperty ;
+        rdfs:label   "inExchangeFor" ;
+        rdfs:domain  vf:Commitment ;
+        rdfs:range   vf:EconomicEvent ;
+        rdfs:comment        "References the economic event that generated the commitment, in cases where based on an agreement, an agent commits to some reciprocity if an event occurs." .
+
+vf:contains  a       owl:ObjectProperty ;
+        rdfs:label   "contains" ;
+        rdfs:domain  vf:EconomicResource ;
+        rdfs:range   vf:EconomicResource ;
+        rdfs:comment        "Used when a stock economic resource contains units also defined as economic resources." .
+        
+        
+vf:quantity a owl:ObjectProperty ;
+        rdfs:label      "quantity" ;
+        rdfs:domain     vf:EconomicResource ;
+        rdfs:range      qudt:QuantityValue ;
+        rdfs:comment    "The current amount and unit of the economic resource." .
+
+vf:affectedQuantity a owl:ObjectProperty ;
+        rdfs:label      "quantity" ;
+        rdfs:domain     vf:EconomicEvent ;
+        rdfs:range      qudt:QuantityValue ;
+        rdfs:comment    "The quantity and unit of the event. This is the quantity that could be used to increment or decrement a resource, depending on the type of resource and type of event." .
+
+vf:currentLocation a owl:ObjectProperty ;
+        rdfs:label   "current location" ;
+        rdfs:domain  vf:EconomicResource ;
+        rdfs:range   vf:Location ;
+        rdfs:comment        "The current place an economic resource is located.  Could be at any level of granularity, from a town to an address to a warehouse location.  Should be mappable." .
+        
+vf:primaryLocation a owl:ObjectProperty ;
+        rdfs:label   "primary location" ;
+        rdfs:domain  vf:Agent ;
+        rdfs:range   vf:Location ;
+        rdfs:comment        "The main place an agent is located, often an address where activities occur and mail can be sent. This is usually a mappable geographic location.  It also could be a website address, as in the case of agents who have no physical location." .
+
+vf:eventStart a owl:ObjectProperty ;
+        rdfs:label     "start" ;
+        rdfs:domain     vf:EconomicEvent ;
+        rdfs:comment   "The starting date and optionally time of the economic event." .
+
+vf:eventDuration a owl:ObjectProperty ;
+        rdfs:label     "duration" ;
+        rdfs:domain     vf:EconomicEvent ;
+        rdfs:comment   "The observed duration of the economic event." .
+
+vf:image a owl:DatatypeProperty ;
+        rdfs:label      "image" .
+
+vf:url a owl:DatatypeProperty ;
+        rdfs:label      "url" .
+
+vf:name a owl:DatatypeProperty ;
+        rdfs:label      "name" ;
+        rdfs:comment    "An informal or formal textual identifier for an object. Does not imply uniqueness." .
+
+vf:note a owl:ObjectProperty ;
```

Comment:
vf:note is declared twice causing vowl json converter to fail. One needs to be removed.